### DISCUSSION TITLE
Resizable start menu

### DISCUSSION THREAD

**Discussion by docquantum** [Open-Shell/Open-Shell-Menu#648] [Ideas]
There should be a way to resize, in the very least vertically, the menu so that more applications can be seen. This *may* require an extensive rewrite of the UI, so it may not be viable.

This is especially helpful for larger monitors where there is a great deal of vertical room to view recent and pinned applications.

**Comment by blackcrack** [comment]
what did you not understand on the Classic Start menu, you have 3 possibilitys
and if you  want add some possibility to add a Startmenu like KDE/Plasma 
then support the group and deliver a code and support the project 
with a 4rd startmenu like you want..

simply making demands is easy....

i have a 42" Monitor TV and it fit's with my Startmenu ...
and you have also the possibility to scale up you WinNT

best

**Comment by bonzibudd** [comment]
Why not just change the amount of `Recent items` that can be shown? That is functionally the same. You can also add more items to the right-hand side to match.

**Reply by docquantum** [reply]
> 
> 
> Why not just change the amount of `Recent items` that can be shown? That is functionally the same. You can also add more items to the right-hand side to match.

That's what I was looking for. I didn't know that it would actually expand the start menu, I had assumed it would create a scroll bar.

Thank you.

Granted, as an *idea*, maybe it would make sense to change that value by allowing the user to vertical resize the menu? Might be too complicated to implement easily. And it would be a distinct design choice that may not be appreciated by others...

It's just something the W10 start menu allows and is both nice and intuitive since, if you're in All Programs, it also expands that view and lets you see more applications. Which is what I was ultimately going for. Same could be said for horizontal resize to allow more space for application names to not overflow if the user so chooses to want it.

**Reply by bonzibudd** [reply]
It’s probably possible. However, since this program is intended to replicate the older behavior of Windows 7/XP which behaves the same as Open-Shell, I don’t think it would be considered as priority.

**Comment by blackcrack** [comment]
Changing the DPI is also an option in the config... imho...
